manifest.yml

```diff
@@ -67,6 +67,11 @@ schema:
    - "https://raw.githubusercontent.com/"
 
 versions:
+  "2.30.3":
+    description: Fix issue causing error when updating parent IDs.
+    changelog:
+      fixed:
```

Comment:
Would be good to briefly mention fix from https://github.com/cognitedata/dotnet-extractor-utils/pull/458 also